File: FMAplication/Domain/DailyTasks/DailyInformationTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using FMAplication.Core;
using FMAplication.Domain.Bases;
using FMAplication.Domain.Sales;
using FMAplication.Enumerations;

namespace FMAplication.Domain.DailyTasks
{
    public class DailyInformationTask : DailyBaseTask
    {
        public string InsightImage { get; set; }
        public string InsightDescription { get; set; }

        public string RequestImage { get; set; }
        public string RequestDescription { get; set; }
    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/DailyTasks/dailyInformationTask.ts
import { Outlet } from "..";
import { Reason } from "../ExecutionReason/ExecutionReason";
import { DailyBaseTask } from "./dailyBaseTask";

export class DailyInformationTask extends DailyBaseTask{
    insightImage: string;
    insightDescription: string;
    requestImage: string;
    requestDescription: string;
}